azure/lb.sh

```diff
@@ -120,6 +120,25 @@ fn azure_lb_addresspool_delete(name, group, lbname) {
 	)
 }
 
+fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
+	out, err <= (
+		az network lb address-pool show
+					--resource-group $resgroup
+					--lb-name $lbname
+					--name $addrpoolname
+					--output json
+					>[2=]
```

Comment:
In this case, when the resource doesn't exists az python client returns status code 2 and prints the message below at stderr:
Item '<address-pool-name>' does not exist on load_balancers 'load-balance-name'

So, @lborguetti and I decided ommit it and return empty string whether resource doesn't exists. We follow a perspective that is not a error, just a resource not found.
Off course, we can talk about and change it. Thank you again